File: frontend/src/models/Posting.ts
export default class Posting {
    id: string;
    transactionId: string;
    acountId: string;
    name: string;
    amount: string;
    comment: string;
    is_comment: boolean;

    constructor(
        id: string,
        transactionId: string,
        accountId: string,
        name: string,
        amount: string,
        comment: string,
        is_comment: boolean
    ) {
        this.id = id;
        this.transactionId = transactionId;
        this.acountId = accountId;
        this.name = name;
        this.amount = amount;
        this.comment = comment;
        this.is_comment = is_comment;
    }
}

File: internal/models/posting.go
package models

import "github.com/google/uuid"

type Posting struct {
	Id            uuid.UUID `json:"id"`
	TransactionId uuid.UUID	`json:"transactionid"`
	AccountId	  uuid.UUID `json:"accountid"`
	Name          string 	`json:"name"`
	Amount        string 	`json:"amount"`
	Comment		  string 	`json:"comment"`
	IsComment	  bool 		`json:"is_comment"`
}
